lib/scripting/vbscript.ne

```diff
@@ -34,27 +34,52 @@ ConstNameValue       -> ExtendedID _ "=" _ ConstExprDef
 OtherConstantsOpt    -> "," _ ConstNameValue                          {% data => data[2] %}
 
 ConstExprDef         -> "(" _ ConstExprDef _ ")"
-                      | "-" __ ConstExprDef
-                      | "+" __ ConstExprDef
+                      | "-" ConstExprDef                              {% data => estree.unaryExpression(data[0], data[1]) %}
+                      | "+" ConstExprDef                              {% data => estree.unaryExpression(data[0], data[1]) %}
                       | ConstExpr                                     {% id %}
 
+SubDecl              -> "Sub" __ ExtendedID MethodArg:* NL MethodStmt:* _ "End" __ "Sub" NL                        {% pp.subDecl %}
+                      | "Sub" __ ExtendedID MethodArg:* InlineStmt "End" __ "Sub" NL                             
+                      | MethodAccessOpt __ "Sub" __ ExtendedID MethodArg:* NL MethodStmt:* "End" __ "Sub" NL     
+                      | MethodAccessOpt __ "Sub" __ ExtendedID MethodArg:* InlineStmt "End" __ "Sub" NL          

```

Comment:
Are the other three matches supported? If not, could you comment them out so it's clear?

Reply:
Maybe a good pattern would be:

- Comment those which don't need a pre-processor (e.g. `# no pp`)
- Comment out those that are not supported / tested

Reply:
Yeah, my point was more either do that or comment it out. We can probably assume that a line without a pre-processor isn't tested or working, but in this case I'd prefer it commented out completely.